lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1450,6 +1450,7 @@ def __init__(
         relation_name: str = DEFAULT_RELATION_NAME,
         alert_rules_path: str = DEFAULT_ALERT_RULES_RELATIVE_PATH,
         recursive: bool = False,
+        skip_alert_topology_labeling: bool = False,
```

Comment:
**question:** Should we name it similar to grafana, e.g. `inject_topology_labels=True`?

**suggestion:** Should probably document that it's not just alert labels but also `expr` qualifiers.